src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ConflictResolution.targets

```diff
@@ -25,6 +25,20 @@ Copyright (c) .NET Foundation. All rights reserved.
           To do this, we look at the files that would be copied local when CopyLocalLockFileAssemblies is true
           -->
       <_LockFileAssemblies Include="@(AllCopyLocalItems->WithMetadataValue('Type', 'assembly'))" />
```

Comment:
It looks like it is.  NuGet.ProjectModel is returning "assembly" for the Type property for this entry in the assets file:

```json
      "runtime.debian.8-x64.runtime.native.System.Security.Cryptography.OpenSsl/4.3.0": {
        "type": "package",
        "runtimeTargets": {
          "runtimes/debian.8-x64/native/System.Security.Cryptography.Native.OpenSsl.so": {
            "assetType": "native",
            "rid": "debian.8-x64"
          }
        }
      },
```

So when we get to this line, it is included in what we pass to the conflict resolution.